action/protocol/context.go

```diff
@@ -263,6 +264,7 @@ func WithFeatureCtx(ctx context.Context) context.Context {
 			CandidateRegisterMustWithStake:          !g.IsToBeEnabled(height),
 			DisableDelegateEndorsement:              !g.IsToBeEnabled(height),
 			SuicideTxLogMismatchPanic:               g.IsToBeEnabled(height),
+			PanicOnDuplicateRevert:                  g.IsToBeEnabled(height),
```

Comment:
default value (false) as the value after hard fork

Reply:
ok, to me using a positive (enable after height) flag name is more natural, which help improve code readability and reduce chance of programming mistake, I think that's more valuable than keeping the default value = false after hard-fork

Reply:
readability is related to https://github.com/iotexproject/iotex-core/pull/4178/files#diff-8d0192f25f9daf6c1bfa70c3b75949444fe5d1d47193e6b7c1210c39a19ed52aR409, but not the default value in the feature context